src/scenarios/shared/startup.py

```diff
@@ -60,10 +61,10 @@ def runtests(self, apptorun: str, **kwargs):
         reportjson = os.path.join(TRACEDIR, 'perf-lab-report.json')
         defaultiterations = '1' if runninginlab() and not uploadtokenpresent() else '5' # only run 1 iteration for PR-triggered build
         startup_args = [
-            self.startupexe,
+            self.startuppath,
             '--app-exe', apptorun,
             '--metric-type', kwargs['startupmetric'], 
-            '--trace-file-name', '%s_startup.etl' % (kwargs['scenarioname'] or '%s_%s' % (kwargs['exename'],kwargs['scenariotypename'])),
+            '--trace-name', '%s_startup' % (kwargs['scenarioname'] or '%s_%s' % (kwargs['exename'],kwargs['scenariotypename'])),
```

Comment:
Since we're losing `.etl` here shouldn't we re-append it somewhere else (in startup.cs perhaps?)

Reply:
The PR will move it to ```WindowsTraceSession.cs```

Reply:
Ahh that already happened in another PR, got it.